app/indexers/curation_concerns/work_indexer.rb

```diff
@@ -8,6 +8,7 @@ def generate_solr_document
         solr_doc[Solrizer.solr_name('member_ids', :symbol)] = object.member_ids
         Solrizer.set_field(solr_doc, 'generic_type', 'Work', :facetable)
         solr_doc[Solrizer.solr_name('suppressed', STORED_BOOL)] = object.suppressed?
+        solr_doc[Solrizer.solr_name('hasRelatedMediaFragment', :symbol)] = object.representative_id
```

Comment:
okay yes both fields are coming through AF for works, but not for fileset. I'm not really sure why, or how to tell what other classes will get it through AF.

Esp. regarding hasRelatedImage, when @tpendragon added it to the indexes_thumbnails service, that propagates to a bunch of places. This is a duplication when it comes to works and I could take it out of the service and put it directly on the fileset indexer but is it also needed in the collection and admin set indexers and if so do you prefer it in the service even though we don't need it for the work?